pkg/kubelet/network/cni/cni.go

```diff
@@ -53,14 +59,18 @@ type cniNetwork struct {
 
 func probeNetworkPluginsWithVendorCNIDirPrefix(pluginDir, vendorCNIDirPrefix string) []network.NetworkPlugin {
 	configList := make([]network.NetworkPlugin, 0)
-	network, err := getDefaultCNINetwork(pluginDir, vendorCNIDirPrefix)
-	if err != nil {
-		return configList
-	}
-	return append(configList, &cniNetworkPlugin{
-		defaultNetwork: network,
+	plugin := &cniNetworkPlugin{
+		defaultNetwork: nil,
```

Comment:
@mikedanese I agree regarding podnetwork pods should fail until cni is configured -- and the network plugin daemonsets should use host-networking to skip this restriction.

However, I may be missing something, so if this works -- ignore me: 

In: 
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet.go#L2470 
wouldn't this always error (and skip pod sync) due to: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet.go#L2920